lib/screens/onboarding_screen/onboarding_screen.dart

```diff
@@ -0,0 +1,184 @@
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body:SafeArea(
+        child: Column(
+          children: [
+            Expanded(
+              child: PageView.builder(
+                itemCount: demo_data.length,
+                controller: _pageController,
+
+                onPageChanged: (index){
+                  setState(() {
+                    _pageIndex = index;
+                  });
+                },
+
+
+                
+              itemBuilder: (context, index) =>Onbordingcontent(
+                image: demo_data[index].image,
+                title: demo_data[index].title,
+                description: demo_data[index].description,
+              ), 
+                  ),
+            ),
+
+            Row(
+              children: [
+               
+                 SizedBox(
+              height: 40,
+              width: 80,
+              child: ElevatedButton(
+                onPressed: () {if(
+                        _pageController.hasClients ){
+                          _pageController.previousPage(duration: Duration(milliseconds: 300 ), curve: Curves.ease);
+                        }} ,
+                style: ElevatedButton.styleFrom(backgroundColor: Colors.green
+                ),
+                child: Text("Back"),
+              )
+            ),
+                
+                   const Spacer(),              
+                ...List.generate(demo_data.length,(index)=>Padding(
+                  padding: EdgeInsets.only(right:4),
+                  child: DotIndicator(isActive: index == _pageIndex),
+                ),
+                
+                ),
+                 const Spacer(),
+                   SizedBox(
+                  height: 40,
+                  width: 80,
+                  child: ElevatedButton(
+                    onPressed: () {
+                      if(
+                        _pageController.hasClients ){
+                          _pageController.nextPage(duration: Duration(milliseconds: 300 ), curve: Curves.ease);
+                        }
+                    } ,
+                    style: ElevatedButton.styleFrom(backgroundColor: Colors.green
+),
+                    child: Text("NEXT"),
+                  )
+                ),
+               
+              ],
+             
+            ),
+           
+          ],
+        ) ,
+      ),
+    );
+  }
+}
+
+class DotIndicator extends StatelessWidget {
+  const DotIndicator({
+    super.key, this.isActive =false,
+  });
+final bool isActive;
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: isActive ? 12:4,
+      width:4,
+      decoration: BoxDecoration(color: Colors.green,borderRadius: BorderRadius.all(Radius.circular(12)),
+       ),
+    );
+  }
+}
+
+class Onboard{
+
+  final String image, title, description;
+
+  Onboard({required this.image, required this.title,required  this.description});
+
+
+}
+
+final List<Onboard>demo_data =[
```

Comment:
Pay attention to the spelling mistakes here in demo_data. Also, Please move the class definition and all the data into a separate file in the "models" folder